tools/unicodetools/com/ibm/rbm/resources/RBManager_sv.properties

```diff
@@ -325,7 +325,7 @@ dialog_preferences_locale_iso=ISO\ Standarder\:
 dialog_preferences_locale_machine=Kompatibel\ utrustning\:
 # @translated true @created 2000-11-10 @modified 2000-11-08 @creator Jared Jackson @modifier Stefan Edlund
 # @comment 
-dialog_preferences_locale_warning=OBS.\ Andringar\ till\ lokalen\ ar\ inte\ aktiva\ tills\ applikatoinen\ aterstartas
```

Comment:
Interesting.... do you know Swedish?

Reply:
not really hahaha I googled it :laughing:

Reply:
:)